stories/getting-started/themes-props-table.tsx

```diff
@@ -1,121 +1,129 @@
 import React from "react";
-import {
-    DefaultCol,
-    DescriptionCol,
-    NameCol,
-    Section,
-    Table,
-} from "../storybook-common/api-table";
+import { ApiTable } from "../storybook-common/api-table";
+import { ApiTableSectionProps } from "../storybook-common/api-table/types";
 
-export const PropsTable = () => (
-    <Table>
-        <tr>
-            <NameCol mandatory>colorScheme</NameCol>
-            <DescriptionCol propTypes={[`"base"`, `"bookingsg"`]}>
-                Specifies the color scheme of the theme
-            </DescriptionCol>
-            <DefaultCol>
-                <span>-</span>
-            </DefaultCol>
-        </tr>
-        <tr>
-            <NameCol mandatory>textStyleScheme</NameCol>
-            <DescriptionCol propTypes={[`"base"`]}>
-                The text style of the theme that will affect headings, body and
-                hyperlinks
-            </DescriptionCol>
-            <DefaultCol>
-                <span>-</span>
-            </DefaultCol>
-        </tr>
-        <tr>
-            <NameCol>options</NameCol>
-            <DescriptionCol propTypes={[`ThemeSpecOptions`]}>
-                The options to customise the color scheme or the text style
-                scheme
-            </DescriptionCol>
-            <DefaultCol>
-                <span>-</span>
-            </DefaultCol>
-        </tr>
+const DATA: ApiTableSectionProps[] = [
+    {
+        attributes: [
+            {
+                name: "colorScheme",
+                mandatory: true,
+                description: <>Specifies the color scheme of the theme</>,
+                propTypes: [`"base"`, `"bookingsg"`],
+            },
+            {
+                name: "textStyleScheme",
+                mandatory: true,
+                description: (
+                    <>
+                        The text style of the theme that will affect headings,
+                        body and hyperlinks
+                    </>
+                ),
+                propTypes: ["base"],
+            },
+            {
+                name: "designTokenScheme",
+                mandatory: true,
+                description: (
+                    <>
+                        The design token scheme that will affect common style
+                        properties throughout components
+                    </>
+                ),
+                propTypes: [`"base"`],
+            },
+            {
+                name: "options",
+                description: (
+                    <>
+                        The options to customise the color scheme, the text
+                        style scheme or the design token scheme
+                    </>
+                ),
+                propTypes: ["ThemeSpecOptions"],
+            },
+        ],
+    },
+    {
+        name: "ThemeSpecOptions",
+        attributes: [
+            {
+                name: "options.color",
```

Comment:
Since it is sitting under the `ThemeSpecOptions` I don't think you require the `option` prefix